### ISSUE TITLE
bad link for customizing

### ISSUE THREAD

**Issue by scenaristeur** [gterrono/houston#360]
Hello, on this page : https://atmospherejs.com/houston/admin, the link for " see Customizing Houston "
wich land to https://github.com/gterrono/houston/blob/master/#customizing-houston does'nt work . 

same thing on https://atmospherejs.com/houston/admin to the link ' Custom Action buttons for collections ' --> https://github.com/gterrono/houston/blob/master/#custom-actions

Thanks!

**Comment by AlexeyMK** [comment]
Good catch! If you'd be up for being a contributor, please submit a PR!
